test/verify/check-machines

```diff
@@ -206,12 +219,44 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         return args
 
     def testBasic(self):
+        self._testBasic()
+
+    def testBasicLibvirtUserUnprivileged(self):
+        # https://libvirt.org/dbus.html#usage
+        busconfig = """
+<?xml version="1.0"?>
+<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+  <policy user="{0}">
```

Comment:
This doesn't make much sense, IMHO. There's no point adding the user to the libvirt group if the rule is on the user instead of the  group. In fedora, /usr/share/dbus-1/system.d/org.libvirt.conf ships this rule by default, which explains why it works OOTB:
```xml
  <policy group="libvirt">
    <allow send_destination="org.libvirt"/>
  </policy>
```

But this rule is present in e. g. ubuntu-stable as well, so *in principle* it ought to work. I'd rather naughty or skip it for now, and file a bug report in the meantime.

Reply:
I can't reproduce locally :/ It fails everywhere apart from fedora images basically.
I skipped it for now in all images apart from fedora.